using-timescaledb/data-retention.md

```diff
@@ -1,15 +1,21 @@
 # Data Retention [](data-retention)
 
-TimescaleDB allows efficient deletion of old data at the chunk level
-using the `drop_chunks` function.
+As your tables grow, it can become cumbersome to keep all that data
+available in the hypertable, so you will need to remove data that are
+no longer relevant. Since data in hypertables are chunked, a normal
+`DELETE` will be very inefficient and TimescaleDB instead provide the
+`drop_chunks` function that is able to quickly drop chunks that are
+not needed any more.
+
+For example:
 
 ```sql
 SELECT drop_chunks(interval '24 hours', 'conditions');
 ```
 
-This will drop all chunks from the hypertable 'conditions' that _only_ include
-data older than this duration, and will _not_ delete any
-individual rows of data in chunks.
+This will drop all chunks from the hypertable `conditions` that
```

Comment:
This is wrong because it changes the meaning of what was there before and is no longer reflecting how `drop_chunks` works.

Note that `drop all chunks ... that include data older than 24 hours` is not correct, because if a chunk includes some data that is older than 24 hours but some that is younger than 24 hours, then that chunk will _not_ be dropped. In other words, if the drop-threshold hits somewhere within a chunk, that chunk will remain in its entirety.

Reply:
Right, an "only" is missing. The original sentence was long and a little hard to read so tried to improve it, but restored the original.